Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM golang:1.8 AS build-env
 
-MAINTAINER Mei Akizuru
+LABEL maintainer="Massimo Virgilio <massimovirgilio@hotmail.com>"
```

Comment:
**I'll continue to maintain this repository and the docker image.**
Of course, you can change here in your repository (which is forked from this repo), but please do not create the pull-request including such change 😢

Reply:
I am sorry, it's the first time I do pull request. I didn't know I wasn't supposed to change the maintainer on my forked repo. Just take the flag change, at the end I only wanted to share some good stuff. Also it was the first time I saw golang, hope I did it right.